cmd/kubeadm/app/phases/upgrade/prepull_test.go

```diff
@@ -116,25 +116,25 @@ func TestPrepullImagesInParallel(t *testing.T) {
 		{
 			name:        "should error out; create failed",
 			p:           NewFailedCreatePrepuller(),
-			timeout:     10 * time.Second,
+			timeout:     PrepullImagesInParallelTimeout,
```

Comment:
You should use `constants.PrepullImagesInParallelTimeout`

Reply:
thanks 🤣